mkt/comm/templates/comm/emails/escalation_reserved_perms_app.html

```diff
@@ -3,8 +3,9 @@
 {% block content %}
 Dear Superheroes,
 
-A developer has uploaded/resubmitted an app that uses prerelease permissions
-(the ones that used to be certified but are now privileged).
+A developer has uploaded/resubmitted an app that uses reserved permissions
+(the ones that used to be certified but are now privileged, or that are only
+ intended for specific partners).
```

Comment:
I wonder can we rephrase to avoid saying 'partners' - to amateurs it may sound like we're excluding them; and to commercial developers that they can pay us to use these permissions.  "specific projects" ?